scripts/searchForm.js

```diff
@@ -0,0 +1,66 @@
+let userInput = "";
+
+class SearchForm {
+  constructor(container) {
+    this.container = container;
+    this.input = null;
+    this.searchBtn = null;
+    this.baseUrl =
```

Comment:
baseURL should be defined once for all of the project. You can achive that with helper.js file, or to declare it in index.html